packages/wallets/src/components/WalletTourGuide/WalletTourGuide.tsx

```diff
@@ -20,12 +28,20 @@ const WalletTourGuide = () => {
     const [addMoreWalletsTransformValue, setAddMoreWalletsTransformValue] = useState('');
     const { isMobile } = useDevice();
 
+    // just because someone clicked button in dtrader and set local storage
+    // does not mean we should run the tour as we might need to wait for the account to be switched
+    const [run, setRun] = useState<boolean>(false);
```

Comment:
I think we can let it infer itself
```suggestion
    const [run, setRun] = useState(false);
```

Reply:
removed :)